index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <div id="logo">
+                <h1>My Logo</h1>
+            </div>
+            <div>
+                <i class="fa fa-bars fa-lg"></i>
+            </div>
+        </nav>
+        <header>
```

Comment:
- [ ] Kindly change the `header` tag to a `section` tag. `section` tags are used to represent a generic standalone section of the document, so it would be ideal to use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more information. 💻 👍